src/bert/utils.py

```diff
@@ -149,3 +149,16 @@ def get_output_dim(self):
             return 2 * self.model.config.hidden_size
         else:
             return self.model.config.hidden_size
+
+    def transplant_LM_head(self, args):
+        """
+        download the masked LM classifier from huggingface bert
```

Comment:
Clean this up to match other fns here: Use a full sentence with capitalization. Otherwise looks good!